### PR TITLE
Update rule about K6

### PR DESCRIPTION
**Pull request by YazhiChen**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️Adam requested in email with subject 'RE: SSW Small SQL database options'. related [Product Backlog Item 88994](https://dev.azure.com/ssw/SSW.Induction/_workitems/edit/88994): ⚡ Performance | Create a new App Service plan

> 2. What was changed?

✏️
Add sample code when you need to conduct the loading test with simulated traffic from different regions. 
Include a graph showing the test results in the K6 Cloud

### CODE REVIEW

File: rules/have-tests-for-performance/rule.md [hunk 1]
```diff
@@ -18,7 +20,7 @@ Typically, there are User Acceptance Tests that need to be written to measure th
 
 <!--endintro-->
 
-**Sample Code:**
+#### Sample Code
 
 ``` cs
 import http from 'k6/http';
```
Comment (GordonBeeming) [inline rules/have-tests-for-performance/rule.md:26]: Don't we need a closing ``` tag for this code?

File: rules/have-tests-for-performance/rule.md [hunk 2]
```diff
@@ -37,6 +39,62 @@ export default function () {
...
+      'status was 200': (r) => r.status === 200
+    });
+    sleep(1);
+  };   
+}
+```
+**Figure: This code uses k6 to test several endpoints by simulating traffic from different regions**
+
+::: good  
+![Figure: Good example - Output the result of simulating traffic from West US to K6 Cloud](RunScriptWithK6Cloud.png)
```
Comment (GordonBeeming) [inline rules/have-tests-for-performance/rule.md:95]: The image name should also use kabab-case

https://www.ssw.com.au/rules/use-dashes-in-urls/

File: rules/have-tests-for-performance/rule.md (hunk 1 above)
Comment (YazhiChen) [reply rules/have-tests-for-performance/rule.md:26]: We have a closing ``` tag for this code

File: rules/have-tests-for-performance/rule.md (hunk 2 above)
Comment (YazhiChen) [reply rules/have-tests-for-performance/rule.md]: it is updated.

**Review by GordonBeeming — APPROVED** [review]

**Pull request merged**